e2e/_suites/ingest-manager/features/ingest_manager.feature

```diff
@@ -13,15 +13,15 @@ Scenario: Deploying an agent
 @start-agent
 Scenario: Starting the agent starts backend processes
   When an agent is deployed to Fleet
-  Then "filebeat" is "started" on the host
-    And "metricbeat" is "started" on the host
+  Then the "filebeat" process is "started" on the host
+    And the "metricbeat" process is "started" on the host
 
 @stop-agent
 Scenario: Stopping the agent stops backend processes
   Given an agent is deployed to Fleet
-  When the agent is "stopped" on the host
-  Then "filebeat" is "stopped" on the host
-    And "metricbeat" is "stopped" on the host
+  When the "agent" process is "stopped" on the host
```

Comment:
With this subtle change we can reuse steps. What I'd like to know is if we can consider the agent a process

Reply:
++ for reusability